superset-frontend/src/dashboard/components/gridComponents/ChartHolder.jsx

```diff
@@ -245,8 +245,8 @@ class ChartHolder extends React.Component {
     let chartHeight = 0;
 
     if (this.state.isFullSize) {
-      chartWidth = document.body.clientWidth - CHART_MARGIN;
-      chartHeight = document.body.clientHeight - CHART_MARGIN;
+      chartWidth = window.screen.width - CHART_MARGIN;
```

Comment:
Hi there, why we use `window.screen.width` instead of `window.innerWidth`(previous implementation for you). The `window.screen.width` looks like it always gets the whole screen width instead of the browser width.

Reply:
I 'm sorry! It's not thoughtful. I change it back